lib/Frontend/ModuleInterfaceLoader.cpp

```diff
@@ -1830,6 +1832,10 @@ InterfaceSubContextDelegateImpl::runInSubCompilerInstance(StringRef moduleName,
   // invocation.
   CompilerInvocation subInvocation = genericSubInvocation;
 
+  // save `StrictImplicitModuleContext`
+  bool StrictImplicitModuleContext =
+      subInvocation.getFrontendOptions().StrictImplicitModuleContext;
```

Comment:
Why do we need to save and restore `StrictImplicitModuleContext` here? It stands out from the rest of frontend options.